packages/edit-site/src/components/template-actions/rename-menu-item.js

```diff
@@ -53,29 +70,33 @@ export default function RenameMenuItem( { template, onClose } ) {
 			// Persist edited entity.
 			await saveSpecifiedEntityEdits(
 				'postType',
-				template.type,
-				template.id,
+				postType,
+				postId,
 				[ 'title' ], // Only save title to avoid persisting other edits.
 				{
 					throwOnError: true,
 				}
 			);
 
 			createSuccessNotice(
-				template.type === TEMPLATE_POST_TYPE
-					? __( 'Template renamed.' )
-					: __( 'Template part renamed.' ),
+				sprintf(
+					// translators: %1$s is a post type label, e.g., Template, Template Part or Pattern. %2$s is the new title.
+					__( '%1$s renamed to "%2$s".' ),
+					POST_TYPE_LABELS[ postType ] ??
+						POST_TYPE_LABELS.wp_template,
+					editedTitle
+				),
 				{
 					type: 'snackbar',
+					id: 'template-rename-success',
 				}
 			);
 		} catch ( error ) {
-			const fallbackErrorMessage =
-				template.type === TEMPLATE_POST_TYPE
-					? __( 'An error occurred while renaming the template.' )
-					: __(
-							'An error occurred while renaming the template part.'
-					  );
+			const fallbackErrorMessage = sprintf(
+				// translators: %s is a post type label, e.g., Template, Template Part or Pattern.
+				__( 'An error occurred while renaming the %s.' ),
+				POST_TYPE_LABELS[ postType ] ?? POST_TYPE_LABELS.wp_template
```

Comment:
This code is used in multiple places would it aid readability to define it once in a meaningful variable e.g. `postTypeLabel`